spec/controllers/admin/users_controller_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe Admin::UsersController, type: :controller do
+  context 'user is not logged' do
+    it 'GET #index' do
+      get :index
+      expect(response.status).to eq(302)
```

Comment:
statuses you can check in function `expect(response).to have_http_status(:success)`

Reply:
i know this syntax, but response will be - (302 - redirect), because of that, that user is not logged in